proto/message.py

```diff
@@ -448,24 +483,49 @@ def to_dict(
             preserving_proto_field_name (Optional(bool)): An option that
                 determines whether field name representations preserve
                 proto case (snake_case) or use lowerCamelCase. Default is True.
-            including_default_value_fields (Optional(bool)): An option that
+            including_default_value_fields (Optional(bool)): Deprecated. Use argument
```

Comment:
same comments as in `to_json`

Reply:
Fixed in https://github.com/googleapis/proto-plus-python/pull/433/commits/0e65d18eefc21ef7b34b9fc12f9aa779affca437